Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.20-bookworm AS builder
```

Comment:
The minimum version supported by FlyDB is 1.18, may be 1.18 is better.

Reply:
done